agent/custom/auto_battle_action.py

```
@@ -68,17 +68,16 @@ def _plan_from_chaldea_share(param: dict) -> BattlePlan | None:
     支持两种形态（显式 ``plan`` 优先级更高，由调用方保证已在此前解析）：
     1. ``chaldea_share``：已解码的 BattleShareData dict（离线注入用）。
-    2. ``chaldea_import_source`` / ``chaldea_import_source_file``：链接/ID/压缩串
-       或本地文件路径（MXU 文件选择器）。本地文件优先，复用 agent/chaldea 的
-       fetch_share_data 下载 + 解码。
+    2. ``chaldea_import_source``：链接/ID/压缩串或本地文件路径（MXU 文件
+       选择框，路径与手填输入已合并为同一字段），复用 agent/chaldea 的
+       fetch_share_data 下载/解码/本地加载。
     """
     share = param.get("chaldea_share")
     if isinstance(share, dict):
         mfaalog.info(f"[auto_battle] 使用离线注入的 chaldea_share")
```

Ruff (0.16.3)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)